### PR TITLE
Removed Coverity warning in collab and cleaned up all files there

### PR DESCRIPTION
**Pull request by oscargus**

### CODE REVIEW

**Comment by oscargus** [comment]
I need to double-check a few things here...

**Comment by oscargus** [comment]
Should be good to go again.

File: src/main/java/net/sf/jabref/collab/ChangeScanner.java [hunk 1]
```diff
@@ -64,6 +65,7 @@
 
     private static final Log LOGGER = LogFactory.getLog(ChangeScanner.class);
 
+    private final double MATCH_THRESHOLD = 0.4;
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/collab/ChangeScanner.java]: why not static?

File: src/main/java/net/sf/jabref/collab/EntryChange.java [hunk 2]
```diff
@@ -85,7 +85,7 @@ public boolean makeChange(BasePanel panel, BibDatabase secondary, NamedCompound
         boolean allAccepted = true;
 
         Enumeration<Change> e = children();
-        for (; e.hasMoreElements();) {
+        while (e.hasMoreElements()) {
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/collab/EntryChange.java]: Maybe use a foreach loop there instead?

**Comment by simonharrer** [comment]
:+1: except two minor comments.

File: src/main/java/net/sf/jabref/collab/ChangeScanner.java (hunk 1 above)
Comment (oscargus) [inline src/main/java/net/sf/jabref/collab/ChangeScanner.java]: No idea. :-)

File: src/main/java/net/sf/jabref/collab/EntryChange.java (hunk 2 above)
Comment (oscargus) [inline src/main/java/net/sf/jabref/collab/EntryChange.java]: Didn't know that one can have a foreach loop for Enumerations (there are these iterator-whiles everywhere in the code), but now I googled and it wasn't that hard...

**Pull request merged**